app/services/glassdoor_service.rb

```diff
@@ -12,9 +12,16 @@ def company(company_name)
   private
 
   def parse(response)
-    results = JSON.parse(response.body)["response"]
+    results = exact_matches(JSON.parse(response.body)["response"])
+
     if results
-      results["employers"].first
+      results.first
```

Comment:
Yes, it was and still is only pulling the first result. The only real change is that the results have now have been limited to only those employers whose name is an exact match.